### PR TITLE
[18.0][MIG] stock_warehouse_calendar: Migration to 18.0

### PR DESCRIPTION
**Pull request by RicardCForgeFlow**
Migration to 18.0

The README has been improved to follow the [OCA guidelines](https://odoo-community.org/read-me-guidelines), and test coverage has been increased.

@ForgeFlow

### CODE REVIEW

File: stock_warehouse_calendar/tests/test_stock_warehouse_calendar.py [hunk 1]
```diff
@@ -0,0 +1,149 @@
...
+
+from odoo import fields
+from odoo.tests.common import TransactionCase
+
+
+class TestStockWarehouseCalendar(TransactionCase):
+    @classmethod
+    def setUpClass(cls):
+        super().setUpClass()
+        cls.env = cls.env(context=dict(cls.env.context, tracking_disable=True))
```
Comment (rousseldenis) [inline stock_warehouse_calendar/tests/test_stock_warehouse_calendar.py]: FYI, you can use https://github.com/odoo/odoo/blob/18.0/odoo/addons/base/tests/common.py#L21

**Review by rousseldenis — APPROVED** [review]
LGTM

File: stock_warehouse_calendar/tests/test_stock_warehouse_calendar.py (hunk 1 above)
Comment (RicardCForgeFlow) [reply stock_warehouse_calendar/tests/test_stock_warehouse_calendar.py]: Hi @rousseldenis , thank you very much for the review. Is it better like this?

https://github.com/OCA/stock-logistics-warehouse/compare/e211862a6943610f2a678ffa44dffc01eb4362e1..efcfd5befe89f7f39209591d8675f98f5f77cd2c``

Comment (rousseldenis) [reply stock_warehouse_calendar/tests/test_stock_warehouse_calendar.py]: @RicardCForgeFlow You can remove `TransactionCase` :sweat_smile:

Comment (RicardCForgeFlow) [reply stock_warehouse_calendar/tests/test_stock_warehouse_calendar.py]: Oups, that's true 😳

**Review by LoisRForgeFlow — APPROVED** [review]
LGTM, nice job with the improved documentation :+1:

**Pull request merged**